examples/user-authz/app/api/login/route.ts

```diff
@@ -0,0 +1,33 @@
+import { api, db } from '@/nile/Server';
+
+export async function POST(req: Request) {
+  // be sure to set the header when redirecting
+  const loginResp = await api.auth.login(req);
+  if (loginResp && loginResp.status >= 200 && loginResp.status < 300) {
+    const headers = new Headers(loginResp.headers);
+    const body = await new Response(loginResp.body).json();
+    const uuid = api.users.uuid.decode(body.id);
+
+    try {
+      const [tenant] = await db('tenant_users')
```

Comment:
all of this can now be done via `rest`, so we should move to that. the `slug` thing would also still be possible, but this *probably* belongs in middleware, but you could also make a utility function, since this is just an example.